test/startservers.py

```diff
@@ -0,0 +1,81 @@
+import atexit
+import os
+import shutil
+import signal
+import subprocess
+import tempfile
+
+
+config = os.environ.get('BOULDER_CONFIG')
+if config is None:
+	config = 'test/boulder-config.json'
+processes = []
+tempdir = tempfile.mkdtemp()
+
+
+def run(path):
+    binary = os.path.join(tempdir, os.path.basename(path))
+    goargs = '-race' if os.environ.get('GORACE') else ''
+    cmd = 'go build %s -o %s ./%s' % (goargs, binary, path)
+    print(cmd)
+    subprocess.check_call(cmd, shell=True)
+    def _ignore_sigint():
```

Comment:
Yes, exiting everything right away if one server crashes is very highly desirable. I've been meaning to figure out how to do it. If you want to integrate that into this change I would be profoundly grateful!